.github/goreleaser.yaml

```diff
@@ -0,0 +1,481 @@
+project_name: gno
```

Comment:
We should be triggering this workflow only if the actor is a member of @gnolang/tech-staff 

`if github.actor == ...`, I'm not sure what the syntax would be for a member of a GH group, or if that's even possible

Reply:
let me check if that is possible

Reply:
@zivkovicmilos 'github.actor' will contain the username, not the group, so we cannot do what you proposed.